File: LocationSB/src/com/gime/locationsb/LocationOperation.java
package com.gime.locationsb;

public class LocationOperation {

	private int locationType;
	private int locationStatus;
	private String phoneNumber;
	private String wechat;
	private String imei;
	private double latitude;
	private double longitude;
	
	public int getLocationType() {
		return locationType;
	}
	public void setLocationType(int locationType) {
		this.locationType = locationType;
	}
	public int getLocationStatus() {
		return locationStatus;
	}
	public void setLocationStatus(int locationStatus) {
		this.locationStatus = locationStatus;
	}
	public String getPhoneNumber() {
		return phoneNumber;
	}
	public void setPhoneNumber(String phoneNumber) {
		this.phoneNumber = phoneNumber;
	}
	public String getWechat() {
		return wechat;
	}
	public void setWechat(String wechat) {
		this.wechat = wechat;
	}
	public String getImei() {
		return imei;
	}
	public void setImei(String imei) {
		this.imei = imei;
	}
	public double getLatitude() {
		return latitude;
	}
	public void setLatitude(double latitude) {
		this.latitude = latitude;
	}
	public double getLongitude() {
		return longitude;
	}
	public void setLongitude(double longitude) {
		this.longitude = longitude;
	}

	
}

File: PHP/LocationSB/LocationOperation.php
<?php
class LocationOperation {
  /* 成员变量 */
  var $imei;
  var $type;
  var $number;
  var $latitude;
  var $longitude;
  var $locationStatus;


  /* 成员函数 */
  function setImei($par){
     $this->$imei = $par;
  }
  
  function getImei(){
     echo $this->$imei . PHP_EOL;
  }
  
  function setType($par){
     $this->$type = $par;
  }
  
  function getType(){
     echo $this->$type . PHP_EOL;
  }
  
  function setNumber($par){
     $this->$number = $par;
  }
  
  function getNumber(){
     echo $this->$number . PHP_EOL;
  }
  
  function setLatitude($par){
     $this->$latitude = $par;
  }
  
  function getLatitude(){
     echo $this->$latitude . PHP_EOL;
  }
  
  
  function setLongitude($par){
     $this->$longitude = $par;
  }
  
  function getLongitude(){
     echo $this->$longitude . PHP_EOL;
  }
  
  
  function setLocationStatus($par){
     $this->$locationStatus = $par;
  }
  
  function getLocationStatus(){
     echo $this->$locationStatus . PHP_EOL;
  }
}
?>





